appintro/src/main/java/com/github/appintro/AppIntroFragment.kt

```diff
@@ -32,6 +33,15 @@ class AppIntroFragment : AppIntroBaseFragment() {
          */
         @JvmOverloads
         @JvmStatic
+        @Deprecated(
+            "`newInstance` is deprecated to support color resources with configuration " +
```

Comment:
```suggestion
            "`newInstance` is deprecated to support color resources instead of color int for configuration changes and dark theme support "
```

Let's try to be a bit more informative